Assignment/solution_student/index.html

```diff
@@ -8,24 +8,88 @@
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
+
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
 
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
-    
-      <header>
-        <h1>FEWD HTML Template</h1>
+
+      <header class="clearfix">
+        <div class="header-content">
+          <div class="logo">Relaxr</div>
```

Comment:
Not a big deal, but I might mark this up as an `h1`, just because it is the title of the site, so it seems like it should be denoted as a heading, rather than a div, which has no meaning.